Raspberry PI - CPU info is not correct

### Discussed in https://github.com/nicolargo/glances/discussions/2615

<div type='discussions-op-text'>

<sup>Originally posted by **jslegers73** December  8, 2023</sup>
Hello,

![image](https://github.com/nicolargo/glances/assets/6212673/c070ab59-9347-496a-a9e2-db292d1033c8)

I have a question what does the 8 mean?</div>

@nicolargo Sorry for that. I have tried it with your instructions on a clean install of the RPI OS. But I get errors :


```shell
pi@rpi-test:~ $ mkdir ~/tmp
cd ~/tmp
git clone -b develop https://github.com/nicolargo/glances.git
cd ~/tmp/glances
git checkout develop
Cloning into 'glances'...
remote: Enumerating objects: 36885, done.
remote: Counting objects: 100% (3869/3869), done.
remote: Compressing objects: 100% (1005/1005), done.
remote: Total 36885 (delta 2612), reused 3575 (delta 2460), pack-reused 33016
Receiving objects: 100% (36885/36885), 40.40 MiB | 23.49 MiB/s, done.
Resolving deltas: 100% (26196/26196), done.
Already on 'develop'
Your branch is up to date with 'origin/develop'.
pi@rpi-test:~/tmp/glances $ cd ~/tmp/glances
make venv
virtualenv -p /usr/bin/python3 venv
make: virtualenv: No such file or directory
make: *** [Makefile:24: venv-full-python] Error 127
pi@rpi-test:~/tmp/glances $ cd ~/tmp/glances
make test
./venv/bin/python ./unittest-core.py
make: ./venv/bin/python: No such file or directory
make: *** [Makefile:67: test-core] Error 127
pi@rpi-test:~/tmp/glances $ cd ~/tmp/glances
make run-webserver
./venv/bin/python -m glances -C ./conf/glances.conf -w
make: ./venv/bin/python: No such file or directory
make: *** [Makefile:267: run-webserver] Error 127

```

I have tried it with python -m venv .env and try the steps but still get the error :

pi@rpi-test:~ $ source .env/bin/activate
(.env) pi@rpi-test:~ $ cd ~/tmp/glances
make venv
virtualenv -p /usr/bin/python3 venv
make: virtualenv: No such file or directory
make: *** [Makefile:24: venv-full-python] Error 127
(.env) pi@rpi-test:~/tmp/glances $

Sorry I can't test it for you.

> But a friend of mine told me I can test it on a docker running bullseye. So I will try that tonight.

I have tried it on a debian bullseye container but still get the file not found error on the develop version :

make venv
virtualenv -p /usr/bin/python3 venv
make: virtualenv: No such file or directory
make: *** [Makefile:24: venv-full-python] Error 127
(.env) pi@rpi-test:~/tmp/glances $

Sorry that I could not test it.

--- a/glances/cpu_percent.py
+++ b/glances/cpu_percent.py
@@ -71,7 +71,7 @@ class CpuPercent:
 
     def __get_cpu_name(self):
         # Get the CPU name once from the /proc/cpuinfo file
-        # Read the first line with the "model name"
+        # Read the first line with the "model name" ("Model" for Raspberry Pi)
         ret = None
         try:
             cpuinfo_file = open('/proc/cpuinfo').readlines()
@@ -79,7 +79,7 @@ class CpuPercent:
             pass
         else:
             for line in cpuinfo_file:
-                if line.startswith('model name'):
+                if line.startswith('model name') or line.startswith('Model'):
                     ret = line.split(':')[1].strip()
                     break
         return ret if ret else 'CPU'

